src/Element/CiviremoteFundingDashboardGroup.php

```diff
@@ -48,6 +50,8 @@ public static function preRenderDashboardGroup(array $element): array {
 
     return [
       '#type' => 'container',
+      '#prefix' => '<div class="civiremote-funding-dashboard-elements">' . $element['#prefix'],
+      '#suffix' => $element['#suffix'] . '</div>',
```

Comment:
While I'm unsure about the semantic implications, an experienced Drupal Core contributor suggests using a `container` render element with the CSS attribute instead of `#prefix` and `#suffix` in [this *Drupal Answers* post](https://drupal.stackexchange.com/a/382). At least that would not put HTML code inside PHP.

Reply:
I've added a comment to the code explaining it.

Reply:
I was trying to say that we could think about an *extra* `container` element around that one which holds the CSS class. What do you think?

Reply:
The elements are now in an extra `container`. Definitely nicer :)